Card/src/main/java/com/paypal/android/card/network/CardAPIClient.kt

```diff
@@ -0,0 +1,42 @@
+package com.paypal.android.card.network
+
+import com.paypal.android.card.Card
+import com.paypal.android.card.CardError
+import com.paypal.android.card.CardResponse
+import com.paypal.android.card.CardResult
+import com.paypal.android.core.APIClient
+import com.paypal.android.core.PayPalJSON
+import org.json.JSONObject
+import java.net.HttpURLConnection.HTTP_OK
+
+class CardAPIClient(private val api: APIClient) {
+
+    suspend fun confirmPaymentSource(orderId: String, card: Card): CardResult {
+        val path = "v2/checkout/orders/$orderId/confirm-payment-source"
+        val body = """
+            {
+                "payment_source": {
+                    "card": {
+                        "number": "${card.number}",
+                        "expiry": "${card.expiry}"
+                    }
+                }
+            }
+        """.trimIndent()
+
+        val httpResponse = api.post(path, body)
+        return if (httpResponse.status == HTTP_OK) {
+            val json = PayPalJSON(httpResponse.body)
+
+            val status = json.optString("status")
+            val id = json.optString("id")
+            val lastDigits = json.optString("payment_source.card.last_digits")
+            val brand = json.optString("payment_source.card.brand")
+            val type = json.optString("payment_source.card.type")
```

Comment:
We could do something like this 👆 . We're mostly pulling strings and numbers from the output I'd imagine. We do something similar in BT iOS. A few convenience wrappers allow us to keep our merchants' number of transitive dependencies small.